libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/PhotoUploadButtonOverlay.java

```diff
@@ -1,55 +0,0 @@
-package net.cyclestreets.views.overlay;
-
-import org.osmdroid.views.MapView;
-
-import android.content.Context;
-import android.content.Intent;
-import android.content.res.Resources;
-import android.graphics.Canvas;
-import android.view.MotionEvent;
-
-import net.cyclestreets.PhotoUploadActivity;
-import net.cyclestreets.view.R;
-
-public class PhotoUploadButtonOverlay extends ButtonOnlyOverlay {
```

Comment:
Is something in place to replace `PhotoUploadButtonOverlay` and `PhotoUploadActivity`?  Or are we removing this functionality?  It's not 100% clear to me.

Reply:
This has been dead for a while - replaced by PhotoUploadFragment